packages/qwik/src/optimizer/core/src/snapshots/qwik_core__test__example_import_attributes.snap

```diff
@@ -0,0 +1,37 @@
+---
+source: packages/qwik/src/optimizer/core/src/test.rs
+assertion_line: 1946
+expression: output
+---
+==INPUT==
+
+
+import { component$, $ } from '@builder.io/qwik';
+import json from "./foo.json" with { type: "json" };
+
+export const Greeter = component$(() => {
+    return json;
+});
+
+== DIAGNOSTICS ==
+
+[
+  {
+    "category": "error",
```

Comment:
this shouldn't error

Reply:
right, I'm trying to upgrade swc which supports `with` in the update-rust branch.

Reply:
awesome